backend/de.metas.payment.revolut/src/main/java/de/metas/payment/revolut/PaySelectionService.java

```diff
@@ -148,7 +148,7 @@ private RevolutPaymentExport toRevolutExportRequest(@NonNull final I_C_PaySelect
 				.accountNo(bpBankAccount.getAccountNo())
 				.routingNo(bpBankAccount.getRoutingNo())
 				.IBAN(bpBankAccount.getIBAN())
-				.BIC(bpBankAccount.getRoutingNo())
+				.SwiftCode(bpBankAccount.getRoutingNo())
```

Comment:
@metas-ts  is this correct?

Reply:
it's tricky..
> don't we have a dedicated SwitfCode/BIC column in BP Bank account?

i think we do, but we don't display it in the bankaccount window (AD_window_ID=540337), and fixing BIC to SwiftCode already goes beyond the scope..
i'll create an issue..